packages/amplify-util-uibuilder/src/commands/utils/notifyMissingPackages.ts

```diff
@@ -2,10 +2,23 @@ import { pathManager, JSONUtilities, $TSContext, $TSAny } from 'amplify-cli-core
 import { printer } from 'amplify-prompts';
 import fs from 'fs-extra';
 import path from 'path';
+import rangeSubset from 'semver/ranges/subset';
 import { extractArgs } from './extractArgs';
-const REQUIRED_PACKAGES: { [key: string]: string } = { '@aws-amplify/ui-react': '*', 'aws-amplify': '*' };
+import { ReactRequiredDependencyProvider } from '@aws-amplify/codegen-ui-react-old';
+import { ReactRequiredDependencyProvider as ReactRequiredDependencyProviderNew } from '@aws-amplify/codegen-ui-react-new';
 
-export const notifyMissingPackages = (context: $TSContext) => {
+const isUpdatedSchema = (schemas: any[]) => {
```

Comment:
why create a duplicate for this function ?
also present in this file: createUiBuilderComponent.ts

Reply:
the implementations are a bit different as is the use case... Soon `isUpdatedSchema` functions won't be needed because all schemas will be updated. This code has a lifespan of just a couple weeks.